app/components/Options/Options.js

```diff
@@ -11,9 +11,8 @@ export default class Options extends Component {
     super();
     this.state = { fontFamily: '', visible: {} };
 
-    Config.groups.map(group => {
+    Config.groups.forEach(group => {
```

Comment:
Here and similar places I just replaced with `.forEach` because it's simply more accurate no? We don't use any return value so `.map` is simply inaccurate, yeah?